api/types/accesslist/convert/v1/accesslist.go

```diff
@@ -143,6 +143,13 @@ func FromProto(msg *accesslistv1.AccessList, opts ...AccessListOption) (*accessl
 		MemberOf:        memberOf,
 	}
 
+	if msg.Status != nil && msg.Status.SelfAccess != nil {
+		accessList.Status.SelfAccess = &accesslist.SelfAccess{
+			OwnershipType:  uint32(msg.Status.SelfAccess.OwnershipType.Number()),
```

Comment:
Why do we need to cast those enum values to numbers here?

Reply:
I figured providing the enum as a `uint32` in the response would be cleaner than a string or other value, since the frontend also has a `MembershipOrOwnershipType` enum it can convert the 0-2 int to... I also didn't see any existing examples of us using PB types for only certain struct fields

Reply:
Ah, that makes sense I suppose. I've never really worked on code that converts stuff from protos into structs that are then encoded to JSON, so I was not familiar with this approach.